src/mining.ts

```diff
@@ -0,0 +1,186 @@
+import { haveEffect, isWearingOutfit, myBuffedstat, visitUrl } from "kolmafia";
+import { $effect, $element, $skill, $stat } from "./template-string.js";
+
+import { chunk, tuple } from "./utils.js";
+import { damageTakenByElement, extractItems, have } from "./lib.js";
+import { get } from "./property.js";
+
+/**
+ * Mines in the Kingdom of Loathing
+ */
+export enum Mine {
+  /** Inside of Itznotyerzitz Mine */
+  ITZNOTYERZITZ = 1,
+  /** Deep Inside the Knob Shaft */
+  KNOB = 2,
+  /** Anemone Mine */
+  ANEMONE = 3,
+  /** The Gummi Mine (Retired, Crimbo 2011) */
+  GUMMI = 4,
+  /** Crimbonium Mine (Retired, Crimbo 2014) */
+  CRIMBONIUM = 5,
+  /** The Velvet / Gold Mine */
+  VOLCANO = 6,
+}
+
+/**
+ * Coordinate system that the Kingdom of Loathing uses for mining.
+ * The first row, first column and last column are all unbreakable.
```

Comment:
It'd be nice to clarify if these start at 0 or 1 and what they range to. You could even use our `Range` utility type!